### PR TITLE
[RateLimiter] Add types to private properties

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Review by wouterj — APPROVED** [review]

File: src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php [hunk 1]
```diff
@@ -22,32 +22,12 @@
...
-
-    /**
-     * @var int the unix timestamp when the current window ends
-     */
-    private $windowEndAt;
-
-    /**
-     * @var bool true if this window has been cached
-     */
-    private $cached = true;
```
Comment (Tobion) [inline src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php:50]: this is true but the constructor sets it to false. this is very odd.

Comment (derrabus) [reply src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php:50]: Indeed. 🤔

Comment (chalasr) [reply src/Symfony/Component/RateLimiter/Policy/SlidingWindow.php:50]: `$cached` becomes true once the window instance has been serialized for the first time. See `__sleep()`.

File: src/Symfony/Component/RateLimiter/Policy/FixedWindowLimiter.php [hunk 2]
```diff
@@ -29,19 +29,8 @@ final class FixedWindowLimiter implements LimiterInterface
...
-
-    /**
-     * @var int seconds
-     */
-    private $interval;
-
-    /**
-     * @var LockInterface
-     */
-    private $lock;
```
Comment (chalasr) [inline src/Symfony/Component/RateLimiter/Policy/FixedWindowLimiter.php]: Did you mean to use promotion for these removed props?

Comment (derrabus) [reply src/Symfony/Component/RateLimiter/Policy/FixedWindowLimiter.php]: No, they're declared inside the trait already. I'm backporting this, I guess.

Comment (derrabus) [reply src/Symfony/Component/RateLimiter/Policy/FixedWindowLimiter.php]: #42144

**Review by Tobion — APPROVED** [review]

**Pull request merged**